docs/docs/how-tos/develop-local-packages.md

```diff
@@ -0,0 +1,74 @@
+---
+id: develop-local-packages
+title: Develop Local Packages on Nebari
+description: How to install a dev package from a local project
+---
+
+# How to Develop Local Packages on Nebari
+
+As a software development platform, Nebari can be used for active package development. Since `conda-store` manages all environments, a dynamic installation of a local package requires additional care (after all, `pip install -e .` is explicitly disallowed for various reasons).
+
+While the setup for developing local packages may require unfamiliar extra steps for new users, the total time for setup is minimal.
+
+## Why can't I just `pip install -e` my package?
+
+Nebari prohibits the standard local installation of packages, e.g. `pip install -e .`. On your personal computer, when you `pip install` a package into a conda environment, it will be placed inside of the conda environment folder structure. However, on Nebari, the environments are managed by `conda-store` in a directory structure which is read-only to users.
+
+For this reason, all pip installs would instead default to the user's `.local` directory. Unfortunately, _all_ environments will automatically pick up _everything_ has been installed in this directory. It is _extremely_ easy to create a situation in which all environments are broken due to a conflict. In fact, its possible to create a situation that causes a user's JupyterLab server to be unable to start. For this reason, local pip installs are prohibited in Nebari.
+
+:::note
+For more information, check out [the docs on installing pip packages](/docs/how-tos/install-pip-packages).
+:::
+
+## Installing local packages
+
+Despite the limitations on local installations, Nebari does provide a mechanism to install a local package currently under development through the use of virtual environments.
+
+### Building the environment
+
+For this example, we'll walk through creating an environment called `myenv` for a user named `myusername`.
+
+Step 1: Use conda-store to build a basic environment that only includes Python called `myenv` in the namespace `myusername` (i.e. your own namespace).
+
+Step 2: Open a terminal in JupyterLab or VS Code
+
+Step 3: list your envs to find the exact spelling: `conda env list`
+
+Step 4: Activate the environment: `conda activate myusername-myenv`
+
+Step 5: Create a virtual environment (venv) in a folder of your choice: `python -m venv .venv_myenv`
+
+Step 6: Activate the venv: `source .venv_myenv/bin/activate`
+
+Step 7: Install your package via pip: `pip install -e .` or `pip install -r requirements.txt` (as needed)
```

Comment:
I think indenting this or making it a bullet point list would help make this easier to read.

Reply:
This is how it looks currently - 
![image](https://github.com/user-attachments/assets/2af74a6b-caa5-4c79-ae61-9ef160d9aff0)